src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/tests/ClickModal.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import ClickModal from '../ClickModal';
+import React from 'react';
+import {mount} from 'enzyme';
+import pretty from 'pretty';
+
+afterEach(() => document.body.innerHTML = '');
+
+test('The modal should initially not be rendered', () => {
+    const body = document.body;
+    const view = mount(
+        <ClickModal clickElement={<button>Open modal</button>}>
+            <p>Modal content</p>
+        </ClickModal>
+    ).render();
+    expect(view).toMatchSnapshot();
+    expect(body.innerHTML).toBe('');
```

Comment:
Shouldn't that contain at least the button with the `Open modal` text? Or am I missing something.